backend.rb

```diff
@@ -57,14 +56,13 @@ class Backend < Sinatra::Base
       origin_proof:  params[:origin_proof]
     )
 
-    authorization_response = purchase.authorize!
-    if authorization_response.success?
+    success, response_data, response_code = purchase.order!
```

Comment:
So @yuval-netanel what would you suggest?
I also think it's safer to have a range of values, but then it says we will continue with the "success" parameter in order to avoid the "calculation" (if it starts with 2...) more than once. (one time in the purchase.rb and another 2 times at backend.rb). What do you say?